.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file should be excluded from the PR

Reply:
Put it on purpose in https://github.com/hopeit-git/hopeit.engine/pull/58/commits/0147a903db7a9ef25a6b0c1c7d8a0d6fbb825de8